### PR TITLE
Updated animate to animateSource

### PR DESCRIPTION
**Pull request by Jawshy**
Resolves https://github.com/AsYetUntitled/Framework/issues/192. 

#### Changes proposed in this pull request: 
- [animate](https://community.bistudio.com/wiki/animate) to [animateSource](https://community.bistudio.com/wiki/animateSource). 
  - `It is recommended that animateSource command is used instead of animate whenever is possible, as it is more efficient and optimized for MP` - [BIKI](https://community.bistudio.com/wiki/) 

#### Todo: 
- [x] Live test.

### CODE REVIEW

**Comment by BoGuu** [comment]
LGTM - no time to test this myself currently

**Comment by Jawshy** [comment]
Tested as of https://github.com/AsYetUntitled/Framework/pull/193/commits/1de16248f1c95a960d3146e08443a894b583234f: 
arma3server.rpt: 
```
2017/01/08, 21:29:11 Error in expression <nimateSource [format ["Door_%1_source", _door], 0];};
_dome setVariable ["locked>
2017/01/08, 21:29:11   Error position: <_door], 0];};
_dome setVariable ["locked>
2017/01/08, 21:29:11   Error Undefined variable in expression: _door
2017/01/08, 21:29:11 File life_server\init.sqf, line 189
```

Repeated three (3) times and between `Starting Altis Life Server Init` and `End of Altis Life Server Init`. 

[/life_server/init.sqf, line 189](https://github.com/Jawshy/Framework/blob/1de16248f1c95a960d3146e08443a894b583234f/life_server/init.sqf#L189): 
```sqf
for "_i" from 1 to 3 do {_dome setVariable [format ["bis_disabled_Door_%1",_i],1,true]; _dome animateSource [format ["Door_%1_source", _door], 0];};
```

File: life_server/init.sqf [hunk 1]
```diff
@@ -186,7 +186,7 @@ private _pos = [[["Altis", _altisArray], ["Tanoa", _tanoaArray]]] call TON_fnc_t
 _dome = nearestObject [_pos,"Land_Dome_Big_F"];
 _rsb = nearestObject [_pos,_vaultHouse];
 
-for "_i" from 1 to 3 do {_dome setVariable [format ["bis_disabled_Door_%1",_i],1,true]; _dome animate [format ["Door_%1_rot",_i],0];};
+for "_i" from 1 to 3 do {_dome setVariable [format ["bis_disabled_Door_%1",_i],1,true]; _dome animateSource [format ["Door_%1_source", _door], 0];};
```
Comment (Jason2605) [inline life_server/init.sqf]: _door should be _i

File: Altis_Life.Altis/core/housing/fn_lockupHouse.sqf [hunk 2]
```diff
@@ -18,7 +18,7 @@ titleText [localize "STR_House_LockingUp","PLAIN"];
 sleep 3;
 
 for "_i" from 1 to _numberOfDoors do {
-    _house animate[format ["door_%1_rot",_i],0];
+    _house animateSource [format ["Door_%1_source", _door], 0];
```
Comment (Jason2605) [inline Altis_Life.Altis/core/housing/fn_lockupHouse.sqf]: _door should be _i

**Comment by Jawshy** [comment]
No issues found as of https://github.com/AsYetUntitled/Framework/pull/193/commits/8f60c60fe445e618a575208cd66f8fa1e8062b9c.

**Pull request merged**